Sources/Shared/Environment/Environment.swift

```diff
@@ -365,6 +365,19 @@ public class AppEnvironment {
 
     public var matter = MatterWrapper()
 
+    public var threadCredentialsSharingEnabled: Bool {
```

Comment:
i know it's kind of mixing different layers, but…how do you feel about putting this in MatterWrapper?

Reply:
Good point, I think in the future we need to separate them, since they are different concerns, but for now I will move it inside MatterWrapper while we don't have more from thread